docs/howto/use_serverless_stack.md

```diff
@@ -0,0 +1,76 @@
+# HOWTO: Use `elastic-package` to test packages using a Serverless project
+
+## Introduction
+
+`elastic-package` supports creating a new Serverless project to assist developers in testing their own packages in these offerings.
+
+**Please note:** Testing packages using this method may result in additional charges.
+
+
+### The `serverless` stack provider
+
+The `serverless` provider facilitates the creation and management of a Serverless project, running the Elastic Agent and other services as local containers using Docker compose.
+
+To use the `serverless` provider, follow these steps:
+1. Obtain an API key from [Elastic Cloud API keys](https://cloud.elastic.co/account/keys).
+2. Set the environment variable `EC_API_KEY` with the obtained API key:
+   ```shell
+   export EC_API_KEY=<api_key>
+   ```
+3. (Recommended) Create a new elastic-package profile for serverless:
+   ```shell
+   elastic-package profiles create serverless
+   elastic-package profiles use serverless
+   ```
+4. Update the profile configuration with the following keys (e.g. `~/.elastic-package/profiles/serverless/config.yml`):
+   ```yaml
+   stack.elastic_cloud.host: https://cloud.elastic.co
+   # supported observability and security types
+   stack.serverless.type: observability
+   stack.serverless.region: aws-us-east-1
+   ```
+
+After completing these steps, create a new Elastic Serverless project by running:
+```sh
+elastic-package stack up -v -d --provider serverless
+```
+
+Once this command finishes successfully, your environment will be ready to use
+any `elastic-package` subcommand (`install`, `dump`, `test`, ...) targeting the Serverless project created with the above command.
+
+To clean up everything, run:
+```sh
+elastic-package stack down -v
+```
+
+**Recommendation:** Ensure that Serverless projects are properly deleted upon completion of testing to avoid any unexpected charges.
+
+Although Serverless projects are accessible through the management UI at [Elastic Cloud Projects](https://cloud.elastic.co/projects),
+those projects created by `elastic-package` must be deleted using the `stack down` command as previously mentioned.
+This procedure ensures that all resources and local configurations are cleaned up correctly, thereby allowing `elastic-package` to function
+correctly in subsequent commands.
```

Comment:
Emphasize to use `elastic-package stack down` and not delete the Serverless project manually from the UI.